joomla/joomla-cms
# [4.0] TinyMCE a11y-checker

### Summary of Changes
I add the plugin https://github.com/instructure/tinymce-a11y-checker to TinyMCE

### Test instruction
1. Apply this pr
2. You need to run 
`npm install`
3. See a button in the toolbar. The symbol is not OK. But this is only a Request for Commands. There are some other issues to work on, if you like this plugin. For example: Add an option to add or remove it ..
![Articles  Edit   test   Administration](https://user-images.githubusercontent.com/9974686/55873991-5b338780-5b91-11e9-96f2-703f8ad20ee3.png)


4. For testing a11y issues you can use this text.
```
<h1>Test Document</h1>
<h4>the document for testing</h4>
<p><img src="http://placekitten.com/640/480" width="300" /></p>
<table width="100%">
<tbody>
<tr>
<td>A</td>
<td>B</td>
<td>C</td>
</tr>
<tr>
<td>1</td>
<td>2</td>
<td>3</td>
</tr>
</tbody>
</table>
<p><img src="https://canvas.instructure.com/images/login/canvas-logo@2x.png" alt="canvas-logo@2x.png" width="300" /></p>
<p style="color: #ccc;">Fusce id ligula rhoncus, consequat augue aliquam, sagittis ligula. Morbi euismod rutrum dignissim. Ut sed elementum mi. Pellentesque condimentum nisl non eros pretium, ut pharetra urna sodales.</p>
<p><a href="https://en.wikipedia.org/wiki/Free_to_Choose"> <img src="https://placekitten.com/480/640" alt="you are free" width="300px" /> </a> <a href="https://en.wikipedia.org/wiki/Free_to_Choose">You are Free!</a> <img src="https://imgs.xkcd.com/comics/visual_field.png" alt="I recently learned something that solved a mystery that had bugged me since childhood--why, when I looked at an analog clock, the hand would sometimes seem to take a couple seconds to start ticking. Google 'stopped clock illusion'." /></p>
<h1>this is a really big heading tag. 51 chars long even. this is a really big heading tag. 51 chars long even. 12345678901234567890</h1>
<p>Time for some lists!</p>
<p>1. List item</p>
<p>2. List item</p>
<p>3. List item</p>
<p>Time for some bullets!</p>
<p>- List item</p>
<p>- List item</p>
<p>- List item</p>
<p>Edge cases:</p>
<p>1. List item</p>
<p>2. <strong>List item</strong><br />Something separated by a break</p>
<p>3. List item</p>
<p>- List item</p>
<p>- List item</p>
<p>- List item</p>
<p>Split breaks:</p>
<p>1. List Item<br />2. List Item</p>
```
5. After you click on the button, the check element is marked blue and you see hints on the right side.
![Articles  Edit   test   Administration(1)](https://user-images.githubusercontent.com/9974686/55874140-cf6e2b00-5b91-11e9-84b6-e80f7c696dfa.png)


### Documentation Changes Required
Yes

### Misc
License is MIT. It should work with Tiny 4 and Tiny 5 - so we could use it in staging, too.
